application/src/js/cms/_scroll_past_sticky_status_banner.js

```diff
@@ -0,0 +1,36 @@
+function ErrorSummaryWithStickyBanner() {
+}
+
+ErrorSummaryWithStickyBanner.prototype.init = function () {
+    var hrefAnchorRegex = /\#(.+)$/;
+    var errorBox = document.querySelector('.flash-message .alert-box.error');
```

Comment:
Guess we'll soon be changing this to `.govuk-error-summary__list`?

Reply:
Sure